content/spin/go-components.md

```diff
@@ -123,7 +123,7 @@ $ tinygo build -wasm-abi=generic -target=wasi -no-debug -o main.wasm main.go
 ```
 
 Before we can execute this component, we need to add the
-`some-random-api.ml` domain to the application manifest `allowed_http_hosts`
+`fermyon.app` domain to the application manifest `allowed_http_hosts`
```

Comment:
I would expect to have to add `random-data-api.fermyon.app`

Reply:
Thanks; updated in the latest commit.